README.md

```diff
@@ -74,78 +75,64 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-
-The PowerToys team is delaying our 0.39 release. We learned a lot but items took longer than expected. We have more we want done before the next release. This longer time will allow us to get in more amazing pull requests by you, the community, to add / improve functionality
-
-### 0.37 - April 2021 Update
-
-Our goals for [v0.37 release cycle](https://github.com/microsoft/PowerToys/issues?q=is%3Aopen+is%3Aissue+project%3Amicrosoft%2FPowerToys%2F19) Video Conference Mute work so we can bring it into the stable branch, general bug fixes, moving Keyboard manager out, and removing the legacy settings app.
+### 0.41 - June 2021 Update
 
-The 0.36 experimental release was released this month as well which includes Video Conference Mute which is based off the 0.35 code base.
+Our goals for [v0.39 release cycle](https://github.com/microsoft/PowerToys/issues?q=is%3Aopen+is%3Aissue+project%3Amicrosoft%2FPowerToys%2F20) and [v0.41 release cycle](https://github.com/microsoft/PowerToys/issues?q=is%3Aopen+is%3Aissue+project%3Amicrosoft%2FPowerToys%2F21) cycle included stability updates and optimizations, general bug fixes, accessibility improvements, and supporting the integration of the new community led project, Awake, which allows Power-Users to now keep their computer awake on demand! 
 
-Our [prioritized roadmap][roadmap] of features and utilities will dictate what the core team is focusing on for the near future. 
-
-#### Highlights from v0.37
+#### Highlights from v0.39 / v0.41
 
 **General**
 
-- PowerToys now requires Windows 10, version 1903 or higher
-- FancyZones editor default launching key is <kbd>Win</kbd>+<kbd>Shift</kbd>+<kbd>`</kbd>
-   - Windows Terminal's new Quake mode will use <kbd>Win</kbd>+<kbd>`</kbd>.  We feel this is a far better use of the keystroke. 
-   - Current PowerToys users can update this in our settings in the FancyZone section.
-- Removed our v1 HTML based settings system
+- New Awake utility added! Power-Users can now keep their computer awake on-demand without having to manage its power settings. A huge thank you to [@dend](https://github.com/dend) for driving the development of this feature. Check out complete guidance and getting started info on [Microsoft Docs](https://aka.ms/PowerToysOverview_Awake)
+- Improved auto-update experience in PowerToys Settings
+- Improved settings layout for radio button groups. Updated images and menu for OOBE. Thanks [@niels9001](https://github.com/niels9001)!
+- Updated general bug report information.
 
-## New Spec - Feedback please!
+### Color Picker
 
-- What is new in PowerToys (SCOOBE) - [Pull Request](https://github.com/microsoft/PowerToys/pull/10978)
+- New fix to prevent the creation of duplicate colors in the selection history. Thanks [@DoctorNefario](https://github.com/DoctorNefario)!
+- Fixed OOBE hotkey description. Thanks [@coc0a](https://github.com/coc0a)!
+- Updated Color Picker GIF for OOBE. Thanks [@niels9001](https://github.com/niels9001)!
```

Comment:
@niels9001
Didn't you improved layout of edit dialogue and add hotkey support?

Reply:
https://github.com/microsoft/PowerToys/pull/11384

This PR you mean?

Reply:
yes

Reply:
> https://github.com/microsoft/PowerToys/pull/11384
> 
> This PR you mean?

@crutkas, @dedavis678

Reply:
Added, thanks!